### ISSUE TITLE
Import all function dependency of Stan function

### ISSUE THREAD

**Issue by athowes** [epinowcast/primarycensored#171]
In working on https://github.com/epinowcast/epidist/pull/426 I've wanted a way to get a `primarycensored` Stan function and all its dependencies. We don't think this is easy / possible but if it were it would be good.

**Comment by seabbs** [comment]
I agree this would be nice. The only way I can think of doing this is recursively searching for functions used in a given function until no more are found. I think this is quite doable if a bit tedious.
